diagnostic.md

```diff
@@ -40,15 +51,16 @@ methods.
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+A DOM is a map of an website with its source html/css/js which can be altered using
+jquery.
```

Comment:
We can use the `document` object to manipulate how the page gets rendered in the browser, but the JS absolutely cannot be altered. That would be dangerous. After having done some Ajax stuff, can you see why?